### DISCUSSION TITLE
Not following a person but getting their updates

### DISCUSSION THREAD

**Discussion by vishalnandagopal** [community/community#86776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/61023404/636c74bc-38ff-4c22-b86e-1074e35a5e0d)

I am not following eliserichards, but when I click the button to see why I am getting the update, it says I am following them.

**Comment by hangshan-boop** [comment]
the previous version is more stable

**Comment by ghost** [comment]
Try these i think this would help..

- Go to the repository or organization associated with "eliserichards."
- Click on the "Watch" or "Star" button if you have starred or watched the repository. Make sure you are not watching the repository.
- Check your email for GitHub notifications related to the user or repository. GitHub sends email notifications for various activities.
- Check your GitHub Following List whether "eliserichards" is present in it or not i did for you but it is not in there...

I too had this problem... So I completely removed to get notified in explore tab from actions of the person I'm following...

Hope this helps...

**Comment by a-jean-andreasian** [comment]
Literally, I'm currently having three users in my feed never interacted with each other/'s repos.
